mxcubeweb/core/components/beamline.py

```diff
@@ -242,16 +241,6 @@ def beamline_abort_action(self, name):
         :param str name: Owner / Actuator of the process/action to abort
 
         """
-        try:
-            cmds = HWR.beamline.beamline_actions.get_commands()
-        except Exception:
-            cmds = []
-
-        for cmd in cmds:
-            if cmd.name() == name:
-                cmd.abort()
-
-        # Annotated command
```

Comment:
Unrelated change? Can this be done outside of this pull request?

Reply:
Ah, yes Ill extract these changes and make another PR

Reply:
Thanks :)